app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt

```diff
@@ -107,7 +108,7 @@ private fun BitwardenContentBlock(
                 ?: Spacer(Modifier.width(16.dp))
         }
 
-        Column {
+        Column(modifier = Modifier.padding(end = 16.dp)) {
```

Comment:
Can we leave this in the spacer?

Reply:
I couldn't get it to work as it was, it was still going to the very edge of the view.  Any ideas why? On iOS we use spacers to space out views easily vs adding a spacer for padding reasons so I was pretty confused here haha.

Reply:
Wanna give this a whirl:

```kotlin
        Column(modifier = Modifier.weight(weight = 1f, fill = false)) {
            Spacer(Modifier.height(12.dp))
            Text(
                text = headerText,
                style = headerTextStyle,
            )
            subtitleText?.let {
                Spacer(Modifier.height(4.dp))
                Text(
                    text = it,
                    style = subtitleTextStyle,
                    color = BitwardenTheme.colorScheme.text.secondary,
                )
            }
            Spacer(Modifier.height(12.dp))
        }
        Spacer(Modifier.width(16.dp))
```

Reply:
worked like a charm.  thanks!